modules/ssl/ssl_private.h

```diff
@@ -118,6 +118,13 @@
 #define MODSSL_HAVE_ENGINE_API 0
 #endif
 
+/* Use OpenSSL 3.x STORE for loading URI keys and certificates starting with
+ * OpenSSL 3.0
+ */
+#if OPENSSL_VERSION_NUMBER >= 0x30000000
+#define MODSSL_USE_OPENSSL_STORE 1
```

Comment:
Nit: I think it should be MODSSL_HAVE_OPENSSL_STORE, _USE_is usually for when we _HAVE_ it but it can disabled by other means.

Reply:
Thanks, it's possibly missing an:
```
#else
#define MODSSL_HAVE_OPENSSL_STORE 0
```
too.

Reply:
right. will fix, too.